### ISSUE TITLE
c1b15b9b causes failure to load

### ISSUE THREAD

**Issue by fooishbar** [amitab/atom-cscope#15]
To reproduce:
1) Have a project open, with cscope.out in the project root
2) Install atom-cscope
3) Invoke 'atom-cscope: Toggle', or 'atom-cscope: Find symbol' from the command menu
4) Exception occurs :(

I have no idea if this is new or not, since I've never used atom-cscope before, but it does seem to be c1b15b9b which has caused this: it's introduced a block of code using 'path' in which 'path' doesn't appear to have ever been defined.

**Atom Version**: 1.6.0-beta7
**System**: "Fedora release
**Thrown From**: [atom-cscope](https://github.com/amitab/atom-cscope) package, v0.14.0
### Stack Trace

Failed to activate the atom-cscope package

```
At path is not defined

ReferenceError: path is not defined
    at Object.content (/home/daniels/.atom/packages/atom-cscope/lib/views/input-view.coffee:25:46)
    at Builder.tag (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:281:19)
    at Function.View.(anonymous function) [as select] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:83:49)
    at Object.content (/home/daniels/.atom/packages/atom-cscope/lib/views/input-view.coffee:22:10)
    at Builder.tag (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:281:19)
    at Function.View.(anonymous function) [as div] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:83:49)
    at Object.content (/home/daniels/.atom/packages/atom-cscope/lib/views/input-view.coffee:10:8)
    at Builder.tag (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:281:19)
    at Function.View.(anonymous function) [as div] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:83:49)
    at Function.module.exports.InputView.content (/home/daniels/.atom/packages/atom-cscope/lib/views/input-view.coffee:9:6)
    at Function.<anonymous> (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:156:31)
    at Function.View.buildHtml (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:123:10)
    at InputView.View [as constructor] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:155:33)
    at new InputView (/home/daniels/.atom/packages/atom-cscope/lib/views/input-view.coffee:8:3)
    at Object.content (/home/daniels/.atom/packages/atom-cscope/lib/views/atom-cscope-view.coffee:15:33)
    at Builder.tag (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:281:19)
    at Function.View.(anonymous function) [as div] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:83:49)
    at Function.module.exports.AtomCscopeView.content (/home/daniels/.atom/packages/atom-cscope/lib/views/atom-cscope-view.coffee:8:6)
    at Function.<anonymous> (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:156:31)
    at Function.View.buildHtml (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:123:10)
    at AtomCscopeView.View [as constructor] (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:155:33)
    at new AtomCscopeView (/home/daniels/.atom/packages/atom-cscope/lib/views/atom-cscope-view.coffee:7:3)
    at Object.module.exports.AtomCscope.attachModal (/home/daniels/.atom/packages/atom-cscope/lib/atom-cscope.coffee:134:27)
    at Object.module.exports.AtomCscope.activate (/home/daniels/.atom/packages/atom-cscope/lib/atom-cscope.coffee:151:6)
    at Package.module.exports.Package.activateNow (/usr/share/atom-beta/resources/app.asar/src/package.js:181:20)
    at /usr/share/atom-beta/resources/app.asar/src/package.js:800:25
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom-beta/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app.asar/src/command-registry.js:241:20)
    at /usr/share/atom-beta/resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/usr/share/atom-beta/resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:183:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/usr/share/atom-beta/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom-beta/resources/app.asar/src/command-registry.js:260:29)
    at /usr/share/atom-beta/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom-beta/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:570:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom-beta/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:378:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/usr/share/atom-beta/resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/usr/share/atom-beta/resources/app.asar/src/window-event-handler.js:3:61)
```
### Commands

```
  2x -3:33.5.0 blur (atom-text-editor.editor.mini.is-focused)
     -3:30.2.0 pane:show-previous-item (div.panels)
     -3:30 blur (div.panels)
     -3:28 command-palette:toggle (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -3:27 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
  2x -3:23.8.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -3:23.1.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -3:23.1.0 blur (atom-text-editor.editor.mini.is-focused)
     -3:23.1.0 atom-cscope:refresh-db (atom-text-editor.editor.vim-mode.normal-mode)
 19x -3:20.6.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:05.4.0 command-palette:toggle (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:04.5.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
     -0:02.8.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:02.8.0 blur (atom-text-editor.editor.mini.is-focused)
     -0:02.8.0 atom-cscope:toggle (atom-text-editor.editor.vim-mode.normal-mode)
     -0:01.0 blur (atom-text-editor.editor.vim-mode.normal-mode.is-focused)
```
### Config

``` json
{
  "core": {
    "audioBeep": false,
    "autoHideMenuBar": true,
    "themes": [
      "one-dark-ui",
      "predawn-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-cscope, v0.14.0
atomatigit, v1.5.5
autocomplete-glsl, v0.2.3
autocomplete-python, v1.7.0
build, v0.55.0
git-diff-details, v0.23.0
git-log, v0.4.1
hidpi, v0.11.0
language-diff, v0.4.0
predawn-syntax, v1.0.2
project-manager, v2.7.6
vim-mode, v0.64.0

# Dev
No dev packages
```

**Comment by amitab** [comment]
Ah I don't believe it! I missed defining path :/
I will have it fixed right now

**Comment by amitab** [comment]
Fixed in 56cb19d176ae3cd974b672a3905dddd27721d94e
Please update the package and have a look. It should be fixed now :)
